src/serialization/sb3.js

```diff
@@ -345,23 +345,25 @@ const serializeBlocks = function (blocks) {
  */
 const serializeCostume = function (costume) {
     const obj = Object.create(null);
-    obj.assetId = (costume.broken && costume.broken.assetId) || costume.assetId;
     obj.name = costume.name;
-    obj.bitmapResolution = costume.bitmapResolution;
+
+    const costumeToSerialize = costume.broken || costume;
```

Comment:
😎

Reply:
yeah I had been itching to do this for a while but it didn't seem worth it for just 1 or 2 properties but now it's basically all of them! =D